.docker/autoline/docker-compose.dev.yml

```diff
@@ -1,74 +1,54 @@
 version: "3.8"
 
-# Using `node_modules` mounting pattern described here:
-# https://burnedikt.com/dockerized-node-development-and-mounting-node-volumes/#prevent-hosts-node_modules-from-being-mounted
-
 services:
   autoline_mariadb:
+    image: mariadb
     container_name: autoline_mariadb
     build:
-      context: ../../
+      context: ./
       dockerfile: database/Dockerfile
     restart: on-failure
     ports:
-      - "3307:3306"
+      - "3306:3306"
```

Comment:
The `3307` is specifically chosen to not conflict with the MySQL database which can be running locally

Reply:
No because it is configuration for staging.